Common/Securities/HasSufficientBuyingPowerForOrderResult.cs

```diff
@@ -40,5 +40,21 @@ public HasSufficientBuyingPowerForOrderResult(bool isSufficient, string reason =
             IsSufficient = isSufficient;
             Reason = reason ?? string.Empty;
         }
+
+        /// <summary>
+        /// Creates a new result indicating that there is sufficient capital for the contemplated order
+        /// </summary>
+        public static HasSufficientBuyingPowerForOrderResult Sufficient()
+        {
+            return new HasSufficientBuyingPowerForOrderResult(true);
+        }
+
+        /// <summary>
+        /// Creates a new result indicating that there is insufficient capital for the contemplated order
+        /// </summary>
+        public static HasSufficientBuyingPowerForOrderResult Insufficient(string reason)
+        {
+            return new HasSufficientBuyingPowerForOrderResult(false, reason);
```

Comment:
Believe these two new methods are unused, think we are using the similar ones added at `HasSufficientBuyingPowerForOrderParameters.cs`

Reply:
Great catch -- I'll re-review usages of any added methods. I think I initially added them to the result yielding:
``` csharp
return HasSufficientBuyingPowerForOrderResult.Insufficient()
```
and then remembered what we did for `BuyingPowerParameters` where the parameters simply have a `Result` method. Applying this pattern  yields what I ended up deciding on:
``` csharp
return parameters.Insufficient();
```

Thanks!